experimental/parser/testdata/lexer/braces/interleaved.proto.stderr.txt

```diff
@@ -1,11 +1,11 @@
 error: encountered unmatched `[` delimiter
   --> testdata/lexer/braces/interleaved.proto:2:5
-   | 
+   |
  2 |     [
    |     ^ expected a closing `]`
  3 | }
    | - closed by this instead
  4 |     ]
    |     - help: perhaps it was meant to match this?
 
-encountered 1 error
+encountered 1 error
```

Comment:
There is no trailing whitespace highlighted on these diff lines. Is the difference a missing trailing newline or something?

Reply:
Yes, trailing newlines disappeared, and git is doing a bad job of showing that. Fixed.